### DISCUSSION TITLE
Models supported for sentiment analysis

### DISCUSSION THREAD

**Discussion by nancyDoss** [jalammar/ecco#79] [Q&A]
Hi, I have been trying to find a model that would best suit my sentiment analysis using ECCO module, also for the demonstration of explainable AI.  I tried using BERT, DistilBert and Roberta, and all returned the same error stating the model was not supported, although it showed up as supported models earlier. Only t5-base and t5-small are supported for sentiment analysis, are they?

**Comment by jalammar** [comment]
> I tried using BERT, DistilBert and Roberta, and all returned the same error stating the model was not supported, although it showed up as supported models earlier. 

The base models should all be supported -- even if finetuned. I'm wondering if these finetuned models have an additional layer so Ecco is not able to attach to the right output neurons. That's why t5 works, it does not require any changes to the architecture.
